src/views/partials/settings.js

```diff
@@ -178,7 +178,7 @@ export const settings = data => {
           <p class='settings-section-info'>${getMessage('settings-deactivate-account-info')}</p>
           <a
             class='settings-link-fxa'
-            href='${AppConstants.FXA_SETTINGS_URL}'
+            href='${AppConstants.NEXT_PUBLIC_FXA_SETTINGS_URL}'
```

Comment:
Does this need `rel="noopener noreferrer"`?

Reply:
Same as [here](https://github.com/mozilla/blurts-server/pull/3326#discussion_r1309910719).